torch/csrc/jit/python/script_init.cpp

```diff
@@ -684,7 +733,16 @@ void initJitScriptBindings(PyObject* module) {
   // NOLINTNEXTLINE(bugprone-unused-raii)
   py::class_<c10::intrusive_ptr<CustomClassHolder>>(m, "Capsule");
 
-  py::class_<Object>(m, "ScriptObject")
+#define DEF_MAGIC(name) \
```

Comment:
There is no need for a macro or code gen here, a simple loop over the names, using a captured string would result in lower compile time and be easier to follow.